app/javascript/app.js

```diff
@@ -52,7 +52,9 @@ export default template({
       let selector = fragment.replace(/:/g, '\\:')
       let statementRow = document.querySelector(selector)
       if (statementRow) {
+        let headerHeight = document.querySelector('.verification-tool__table th').offsetHeight;
         statementRow.scrollIntoView()
+        window.scrollBy(0, -headerHeight);
```

Comment:
If you wanted to, you could do this in one step (without the `scrollIntoView` followed by a `scrollBy`)…

```js
let statementRow = document.querySelector(selector)
let headerRow = document.querySelector('.verification-tool__table th')
if (statementRow) {
  window.scroll({
    top: statementRow.offsetTop - headerRow.offsetHeight
  })
}
```

It’s not much shorter, but maybe it’s simpler?

Reply:
Yeah, I like that better - the two scrolls does seem a bit silly.

Reply:
Although, as discussed in Slack, `offsetTop` is relative to the parent so now quite the right thing. And the equivalent that works in a good range of browsers is something like:

```
(window.pageYOffset || document.body.scrollTop || document.documentElement.scrollTop) + statementRow.getBoundingClientRect().top - headerRow.offsetHeight
```

... so I think I'll stick with the version with two scrolls since it seems cleaner.